Empire/Empire.cpp

```diff
@@ -1118,7 +1118,7 @@ void ProductionQueue::Update() {
             std::tie(item_cost, build_turns) = queue_item_costs_and_times[key];
             float total_item_cost = item_cost * element.blocksize;
 
-            float additional_pp_to_complete_element = total_item_cost * (1.0f - element.progress); // additional PP, beyond already-accumulated PP to finish this element once
+            float additional_pp_to_complete_element = total_item_cost * (element.remaining - element.progress); // additional PP, beyond already-accumulated PP to finish this element once
```

Comment:
I just realized, perhaps part of the issue had been that the original comment reference to ```build all items``` was, by itself, a bit ambiguous about whether it meant build just the entire block of items, or build all repeats of the block (it was meant to be the latter, and I will double check that I am being more clear  in similar cases in the later PR I submit).   So I should also amend the comment to make that more clear.